src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -230,8 +230,29 @@ protected async Task ApplyTags(string functionArn)
             }
         }
 
-        protected async Task UpdateConfigAsync(GetFunctionConfigurationResponse existingConfiguration, string dotnetSharedStoreValue)
+        /// <summary>
+        /// Reverts the lambda configuration to an earlier point, which is needed if updating the lambda function code failed.
+        /// </summary>
+        /// <param name="existingConfiguration"></param>
+        /// <exception cref="LambdaToolsException"></exception>
+        protected async Task RevertConfigAsync(GetFunctionConfigurationResponse existingConfiguration)
```

Comment:
Thinking this is only called if there was some error. If this function has an error we probably don't want this method's error being the one that the user sees instead of the original error.

What you do you think about renaming this `AttemptRevertConfigAsync` and any errors are just logged and we catch all exceptions.